src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -195,10 +196,11 @@ public void canOverrideRenewalWhenItemIsNotRenewableAndNewDueDateIsNotSpecified(
     UUID nonRenewablePolicyId = loanPoliciesFixture.create(nonRenewablePolicy)
       .getId();
 
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(nonRenewablePolicy)` and remove `UUID nonRenewablePolicyId`?

Reply:
Done.